### ISSUE TITLE
Created note is anonymous, even if user is logged in

### ISSUE THREAD

**Issue by FinixFighter** [organicmaps/organicmaps#1386]
Hi! I created a note via Organic Maps by clicking on a shop->"Edit place"->"Place does not exist", then I wrote something and clicked "Send". The note has been created but it was an anonymous note (https://www.openstreetmap.org/note/2885034), instead it should appear as created by me, since I was logged in (in fact I can see my data if I go in "Settings"->"OpenStreetMap Profile").

**Comment by FinixFighter** [comment]
More than one year is passed and I didn't have this problem anymore. Maybe this is fixed and can be closed?

**Comment by biodranik** [comment]
Does it mean that all notes you're creating now are NOT anonymous?

**Comment by FinixFighter** [comment]
> Does it mean that all notes you're creating now are NOT anonymous?

Yes, exactly. I didn't count how many notes I've created since then, but I can certainly say they're more than 50, and never had this issue anymore.

**Comment by biodranik** [comment]
Interesting. What if you're not logged in? Is an anonymous note created then?

**Comment by FinixFighter** [comment]
> What if you're not logged in? Is an anonymous note created then?

No, I can't create a note if I'm not logged in. OM brings you to the Log In page.

**Comment by biodranik** [comment]
@FinixFighter what if you add a note and cancel log in? Or add a note while offline? Won't it be uploaded as an anonymous one?

**Comment by FinixFighter** [comment]
> @FinixFighter what if you add a note and cancel log in?
I created a note, then logged out, but the note has not been created. It has been created exactly when I logged in again.
https://www.openstreetmap.org/note/3954262

**Comment by FinixFighter** [comment]
> @FinixFighter Or add a note while offline? Won't it be uploaded as an anonymous one?

No, it has just been uploaded when I came back online (after some time, maybe when I restarted the app; in fact I noticed that edits almost always go up when I restart the app).
https://www.openstreetmap.org/note/3954265

**Comment by matkoniecz** [comment]
https://www.openstreetmap.org/note/3968292 - another case of anonymous note created by logged in user - who also complains that their POIs added in OM are not reaching OSM

**Comment by FinixFighter** [comment]
Unfortunately I confirm that problem is still here.
One more case: https://www.openstreetmap.org/note/4246851 (created by me via OrganicMaps while logged in)

**Comment by map-per** [comment]
> Unfortunately I confirm that problem is still present.

Did you do that test with the old or with the new 2025.5.11-3-Google Organic Maps version?

**Comment by FinixFighter** [comment]
> > Unfortunately I confirm that problem is still present.
> 
> Did you do that test with the old or with the new 2025.5.11-3-Google Organic Maps version?

I'm sorry but I'm not 100% sure, even though I think it was the old one. I should have checked the version when reporting, my bad. I'll continue to check if this still occurs and in case I'll report back with new info.

**Comment by map-per** [comment]
Closing this issue as no anonymous notes are updated to OSM anymore.

(Recent nodes can be viewed with [NotesReview](https://ent8r.github.io/NotesReview/?view=list&query=%23organicmaps&limit=250&status=all))
